metrics/topsql.go

```diff
@@ -0,0 +1,43 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Top SQL metrics.
+var (
+	RegisterSQLFail = prometheus.NewCounter(
```

Comment:
```suggestion
	RegisterSQL = prometheus.NewCounter(
```
How about use a single counter and different labels like `success`/`dropped` to distinguish different results count for the common function `RegisterSQL`?

Reply:
done.